config/routes.rb

```diff
@@ -6,5 +6,7 @@
   root 'welcome#index'
 
   get 'register', to: 'users#new', as: :new_user
-  resources :users, only: [:create, :show]
+  resources :users, only: [:create, :show] do
+    get 'discover', to: 'user#discover'
```

Comment:
I think we will need to update this to "users#discover" when we go to make this page and use the controller action - but I'm ok making that change when we get to it for the sake of merging this in now.